File: Solo Learn/Fácil/Aerobarco/hovercraft.py
vendas = int(input())

#producao de 10 hovecraft
gastos = 2 * 10 ** 6 * 10 

#seguro
gastos += 1 * 10 ** 6 
lucro = vendas * 3 * 10 ** 6



if lucro == gastos:
   print("Broke Even")

elif lucro < gastos:
   print("Loss")

elif lucro > gastos:
   print("Profit")

File: Solo Learn/Fácil/Aerobarco/hovercraft.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int vendas = scanner.nextInt();
        int gastos = 2 * (int) Math.pow(10, 6) * 10;
        gastos += 1 * (int) Math.pow(10, 6);
        int lucro = vendas * 3 * (int) Math.pow(10, 6);

        if (lucro == gastos) {
            System.out.println("Broke Even");
        } else if (lucro < gastos) {
            System.out.println("Loss");
        } else if (lucro > gastos) {
            System.out.println("Profit");
        }
    }
}
